### PR TITLE
[MINOR]remove deprecated attribute representation

### PR DESCRIPTION
**Pull request by hungple**
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

**Comment by hungple** [comment]
I have removed deprecated attribute "representation", and used "slots" instead.

**Comment by bdwyer2** [comment]
Jenkins, ok to test
Jenkins, add to whitelist

**Review by bdwyer2 — APPROVED** [review]

**Comment by bdwyer2** [comment]
@hungple looks good thanks!

Do you think you could also fix the spelling of "_pacakge_" in `README.md` with this PR?

**Comment by hungple** [comment]
@bdwyer2 I have fixed the spelling of "pacakge" in README.md and 2 other files.

File: R4ML/R/util.io.R [hunk 1]
```diff
@@ -258,7 +258,7 @@ r4ml.read.csv <- function(
   stringsAsFactors_val <- ifelse(stringsAsFactors, "true", "false")
   inferSchema_val <- ifelse(inferSchema, "true", "false")
   
-  # old way of reading using the external pacakge com.databricks.spark.csv
+  # old way of reading using the external package com.databricks.spark.csv
```
Comment (bdwyer2) [inline R4ML/R/util.io.R]: thanks!

**Pull request closed without merging**